app/graphql/types/query_type.rb

```diff
@@ -33,6 +33,18 @@ def partners_by_tag(tag_id:)
     end
   end
 
+  module ArticleQueries
+    def self.included(klass)
+      klass.field :all_articles, [ArticleType] do
+        description 'Return news articles from all sites for all partners'
```

Comment:
"Return all news articles across the whole site"?

Reply:
no longer applicable